src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int figures = 5;
+    private static final int maxSize = 99;
+    private static final String defaultColor = "white";
```

Comment:
You already have enum WHITE

Reply:
+
I've just added WHITE color in enum!)